manifests/init.pp

```diff
@@ -66,4 +69,32 @@
       }
     }
   }
+
+  if $gsutil_install {
+    # TODO: Windows support.
```

Comment:
https://cloud.google.com/storage/docs/gsutil_install?hl=en#windows
installation on windows uses the same archive, but different paths
and python installation on windows isn't guaranteed, so that's another issue, and so might the execution be

however, given that this thing requires python 2.7, which is EOL, a safe execution on any platform is not guaranteed

Reply:
https://github.com/GoogleCloudPlatform/gsutil/issues/29#issuecomment-525063367

Reply:
would be cool if they updated their documentation

Reply:
Hi @igalic -- totally agreed that the doc on Google's side could use some work (this is seemingly systemic for them). Are we considering this a blocker for acceptance of this PR, or is there something additional I can provide to help shepherd this through?

Thanks very much for the review.

Reply:
it was a remark on my approving of this pr

so i think with enough eyes on this now, we can hit the merge button and see what happens *after* that